addOns/oast/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Added 
+-A context menu to paste payloads from all the supported OAST services. 
```

Comment:
A space needs to be added after the dash, otherwise it will not render correctly.